2.0.0/amd64/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN apt-get update && \
       netbase \
       unzip \
       wget \
+      fontconfig \
```

Comment:
Please do not esid the Dockerfiles directly. Add the fontconfig in the update.sh script in tha mein folder and run it with ./update.sh.This will create the Dockerfiles for you. Try to insert it in an alphabetical order.